projects/kubernetes/cloud-provider-vsphere/1-28/GIT_TAG

```diff
@@ -0,0 +1 @@
+v1.27.0
```

Comment:
same question, im assuming they dont have a 1.28 release?

Reply:
No 1.28 for vsphere but there is one for aws, which I'll add shortly.